### PR TITLE
(WIP) Fields terminology

### PR DESCRIPTION
**Pull request by Freakachoo**
According to issue https://github.com/curiefense/curiefense/issues/552 change a list of field names.

# Changelog
Old Name | New name | Where | CurieConf models
-- | -- | -- | --
Notes | Description | Global Filters (Tag Rules)Flow Control Policies (Flow Control)Rate Limit RulesWAF Rules | m_tagrulem_flowcontrolm_limitm_wafrule
ttl | Time Frame | Flow Control Policies (Flow Control)Rate Limit Rules | m_flowcontrolm_limit
ttl (for ban action only) | Ban Duration | Flow Control Policies (Flow Control)Rate Limit Rules | m_flowcontrolm_limit(UI, when sending from client-side and CurieProxy)
bypass | passthrough | ACL Profiles (ACL Policies) | m_aclpolicy
Exclude WAF Rule | Ignore Rules | WAF Profiles (WAF Policies) | m_wafpolicy(UI, when sending from client-side and CurieProxy)
Mask? | Mask | WAF Profiles | m_wafrule (UI only)
Key (part of Count By fields) | Name | Flow Control Policies (Flow Control) Rate Limit Rules | hovertip on the field in UI and maybe usage in client-side code. No changes on server-side.

### CODE REVIEW

**Review by flaper87 — CHANGES_REQUESTED** [review]
@Freakachoo can you please squash this into fewer commits? 

Git log will be massive and hard to parse, even if we squash it through GH

**Comment by Freakachoo** [comment]
@flaper87 it's done now, but a bit messy with the `parent` commit, sorry :))

**Comment by flaper87** [comment]
@Freakachoo thanks, no worries! Just to confirm, will this break any of the existing APIs? Or is it an internal change only?

**Comment by Freakachoo** [comment]
> @Freakachoo thanks, no worries! Just to confirm, will this break any of the existing APIs? Or is it an internal change only?

@flaper87 shouldn't break the existing API, but it's changing configs jsons.

**Comment by flaper87** [comment]
> > @Freakachoo thanks, no worries! Just to confirm, will this break any of the existing APIs? Or is it an internal change only?
> 
> @flaper87 shouldn't break the existing API, but it's changing configs jsons.

Let's work on a tool to help migrate the configs, as a config change is a breaking change for users that have deployed curiefense.

**Review by flaper87 — APPROVED** [review]
